### PR TITLE
Create several modal of user page

### PR DESCRIPTION
**Pull request by yumeR07**
＊Create "edit meeting" page
Meeting Controller / web.php / reserved.show_details.blade.php / reserved.edit.blade.php(new file)

＊Create "delete meeting" modal
 MeetingController /web.php / reserved.show_details.blade.php / reserved.modals.delete.blade.php(new file)

＊ Create "cancel meeting" modal
MeetingController / web.php / reserved.show_details.blade.php / reserved.modals.cancel_meeting.blade.php(new file)

＊Create "cancel event" modal
EventController / web.php / reserved.show_details.blade.php / reserved.modals.cancel_event.blade.php(new file)

＊Create "join meeting" modal
reserved.show_details.blade.php / reserved.modals.join.blade.php(new file)

＊Code cleanup
top.blade.php / edit.blade.php / research.show.blade.php / reserved.show.blade.php /

### CODE REVIEW

File: app/Http/Controllers/EventController.php [hunk 1]
```diff
@@ -83,6 +83,14 @@ public function destroyAuthParticipant($event_id)
         $participant = Participant::where('email', Auth::user()->email)->first();
         $participant->joinEvents()->where('event_id', $event_id)->detach();
         return redirect()->route('events.show', $event_id);
+
+    }
+  
+    public function cancel(Event $event)
+    {
+        $participant = Participant::where('email', Auth::user()->email)->first();
+        $participant->joinEvents()->detach($event);
```
Comment (raymonkredo) [inline app/Http/Controllers/EventController.php:92]: $this should be `detach($event->id)`

Comment (raymonkredo) [reply app/Http/Controllers/EventController.php:92]: and also please remove the entire
```
 public function __construct(Level $level, Category $category, Meeting $meeting, Room $room){
        $this->level = $level;
        $this->category = $category;
        $this->meeting = $meeting;
        $this->room = $room;
    }
```

if you use something like `$this->event->all()` you can simply replace that with `Event::all()` and its shorter

File: app/Http/Controllers/MeetingController.php [hunk 2]
```diff
@@ -31,22 +31,68 @@ public function create(){
...
+    }
+
+    public function delete(Meeting $meeting){
+        $meeting->forceDelete();
+        return redirect()->back();
+    }
+
+    public function cancel(Meeting $meeting){
+        $user = Auth::user();
+        $user->joinMeetings()->detach($meeting);
```
Comment (raymonkredo) [inline app/Http/Controllers/MeetingController.php:94]: I think it should be `$meeting->id`

Comment (yumeR07) [reply app/Http/Controllers/MeetingController.php:94]: I fixed detach($meeting) to detach($meeting->id)

File: app/Http/Controllers/EventController.php (hunk 1 above)
Comment (yumeR07) [reply app/Http/Controllers/EventController.php:92]: I fix  detach($event) to detach($event->id).

 "public function __construct 〜〜" is  in MeetingController.
I delete this parts and replace with like Event::all().

File: app/Http/Controllers/MeetingController.php [hunk 3]
```diff
@@ -31,22 +31,68 @@ public function create(){
...
         ]);
 
-        $this->meeting->user_id = Auth::user()->id;
-        $this->meeting->title = $request->title;
-        $this->meeting->date = $request->date;
-        $this->meeting->start_at = $request->start_at;
-        $this->meeting->room_id = $request->room_id;
-        $this->meeting->level_id = $request->level_id;
-        $this->meeting->category_id = $request->category_id;
+        $this->meeting->user_id      = Auth::user()->id;
```
[the remark this answers is not in the document]
Comment (yumeR07) [reply app/Http/Controllers/MeetingController.php:42]: I changed this one and added $fillable in MeetingModels.

**Review by Ditzler01 — APPROVED** [review]

**Review by raymonkredo — APPROVED** [review]

**Pull request merged**